### ISSUE TITLE
Add email validation to sign in + registration email inputs

### ISSUE THREAD

**Issue by merlinoa** [Tychobra/polished#117]
```
# From https://www.nicebread.de/validating-email-adresses-in-r/
is_valid_email <- function(x) {
  grepl("^\\s*[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,}\\s*$", as.character(x), ignore.case=TRUE)
}
```

**Comment by phoward38** [comment]
Completed w/ 7afb606098b72cbcf87d47ee9610d1c965061129

- **NOTE:** Still need to add `autofill` fix w/ `form` updates for different browsers

**Comment by phoward38** [comment]
Fixed `autofill` w/ 50608a2afe388d92f65211e3525acad37c7f1d14

**NOTE:** Also fixed `sign_in_module_2` logic for `Sign In`
